### ISSUE TITLE
SSL certificate for conda.io expired today

### ISSUE THREAD

**Issue by gouline** [conda/conda#11791]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

Not sure if this is the right forum to report an issue with your documentation website, but SSL certificate for conda.io expired today (September 2, 2022).

Visiting this URL (still cached in search engines) raises an HTTPS browser error https://conda.io/projects/conda/en/latest/index.html

However, this URL (likely more correct) has a valid certificate https://docs.conda.io/projects/conda/en/stable/index.html

**Comment by Haaroon** [comment]
Yes I too am having this bug which is causing my web browser not load conda.io unless i add it as a security exception
